app/components/partner/client-uploads-stat-chart/index.hbs

```diff
@@ -48,13 +48,16 @@
               class='button'
               type='button'
               local-class='option {{if
-                (eq this.currentTimeline.key option.key)
```

Comment:
This should still be key, as the point of key is to do object matching.

Reply:
Done